client/src/components/settings/Username.js

```diff
@@ -152,7 +152,10 @@ class UsernameSettings extends Component {
     if (!validating && isValidUsername) {
       return (
         <FullWidthRow>
-          <Alert bsStyle='success'>Username is available</Alert>
+          <Alert bsStyle='warning'>
+            Username is available <br/> NOTE: Changing your username will have
```

Comment:
I do not this this is accurate. This should simply say, 

> Changing your username will also change the URL to your profile and your certifications.

That should be enough.

Reply:
Better now?